diff --git a/src/MainWindow.vala b/src/MainWindow.vala
index 8477464..eb2e521 100644
--- a/src/MainWindow.vala
+++ b/src/MainWindow.vala
@@ -50,6 +50,9 @@ namespace Notejot {
                     height_request: 450,
                     width_request: 450);
 
+            Gtk.Settings.get_default().set_property("gtk-theme-name", "elementary");
+            Gtk.Settings.get_default().set_property("gtk-icon-theme-name", "elementary");        
+
             var actions = new SimpleActionGroup ();
             actions.add_action_entries (action_entries, this);
             insert_action_group ("win", actions);